std/haxe/macro/Expr.hx

```diff
@@ -1011,12 +1011,18 @@ class Error extends Exception {
 	**/
 	public var pos:Position;
 
+	/**
+		Child error messages, if any.
+	**/
+	public var childErrors:Array<Error>;
```

Comment:
Should we really make this public API? That means we can't change it later, and I'm not 100% convinced that this is the data structure we ultimately want to go for.

Reply:
We can make it private and use `@:privateAccess` in our tests if you prefer